hail_search/web_app.py

```diff
@@ -8,6 +8,9 @@
 
 logger = logging.getLogger(__name__)
 
+MACHINE_MEM = 128
```

Comment:
should we use something like `psutil.virtual_memory().total` so that this is dynamic in case we have to also switch out the instance size?

Reply:
I'd love to not have to introduce runtime python requirements, since we currently only use whats packaged with the default docker image. We can make this an env variable so its more easily configurable

Reply:
alright.  this is ok by me